styles.css

```diff
@@ -108,3 +108,337 @@ li {
+  padding: 12px;
+  width: max-content;
+  height: 48px;
+  background: #36b37f;
+  border-radius: 4px;
+  text-decoration: none;
+  position: relative;
+  left: 80px;
+  top: 23px;
+  margin-bottom: 210px;
+  border-color: transparent;
+}
+
+#button_resume p {
+  height: 24px;
+  font-family: 'Inter', sans-serif;
+  font-weight: normal;
+  font-weight: 500;
+  font-size: 18px;
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  flex-direction: row;
+}
+
+.skills_container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  padding: 1px;
+  gap: 24px;
+  width: 327px;
+  height: 1106px;
+  top: 15px;
+  margin-top: 25px;
+}
+
+.card_skills {
+  width: 327px;
+  height: 352px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  padding: 50px 0;
+}
+
+#icons {
+  width: 80px;
+  height: 70px;
+  padding: 80px 60px;
+  position: relative;
+  bottom: 55px;
+}
+
+.skill_h2 {
+  width: 337px;
+  height: 32px;
+  font-family: 'inter', sans-serif;
+  font-style: normal;
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 32px;
+  text-align: center;
+  color: #344563;
+  position: relative;
+  bottom: 50px;
+}
+
+.stack_skills1 ul {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  align-content: center;
+  padding: 0 30px;
+  width: 271px;
+  height: 40px;
+  justify-content: space-between;
+  text-align: center;
+  position: relative;
+  bottom: 25px;
+}
+
+.card_skills ul li {
+  font-family: 'inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 14px;
+  line-height: 14px;
+  letter-spacing: 0.03em;
+  color: #36c37e;
+}
+
+.stack_skills2 ul {
+  display: flex;
+  flex-direction: row;
+  align-content: center;
+  align-items: center;
+  text-align: center;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 266px;
+  height: 88px;
+  padding-right: 16px;
+  position: relative;
+  bottom: 35px;
+  right: 10px;
+}
+
+.stack_skills2 ul li {
+  line-height: 70px;
+}
+
+.frame {
+  display: flex;
+  flex-direction: row;
+  text-align: center;
+  gap: 40px;
+  position: relative;
+  left: 40px;
+}
+
+.stack_skills3 ul {
+  width: 290px;
+  height: 88px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  gap: 15px;
+  line-height: 100px;
+  padding-left: 16px;
+  position: relative;
+  bottom: 35px;
+}
+
+.frame_1 {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  left: 110px;
+}
+
+#button:visited {
+  background-color: #008552;
+}
+
+#button:hover {
+  background-color: #36b37f;
+}
+
+#button:active {
+  background-color: #36b37f;
+}
+
+#button_resume:visited {
+  background-color: #008552;
+}
+
+#button_resume:hover {
+  background-color: #36b37f;
+}
+
+#button_resume:active {
+  background-color: #36b37f;
+}
```

Comment:
- [ OPTIONAL ] The styles you applied to the buttons here are not working as expected so please rectify this issue as well. The hover and active effects don't work at all but you need to apply these to improve user interactivity and also so your design can follow the design guidelines in the Figma template :+1: 
- Your design on **hover** and **pressed/active**
![image](https://user-images.githubusercontent.com/102699958/212458874-3888473f-b056-49c7-ba59-489747d335f0.png)
- The Figma design on **hover** and **pressed/active**
![image](https://user-images.githubusercontent.com/102699958/212458915-49d8a5ec-9236-4a0e-a80e-ed6f7e72a7f6.png)